index.js

```diff
@@ -0,0 +1,80 @@
+// function takes in an integer and outputs a random integer between 0 & input
+const randomInt = (max) => {
+  return Math.round(Math.random() * (max))
+}
+
+// function accepts a integer of the size of the gameboard's sides
+// it outputs a 2-dimensional array, initialized with all `0` values
+// `0` represents a dead cell, `1` represents an alive cell
+const createBoard = sideLength => {
+  const board = []
+  for (let i = 0; i < sideLength; i++) {
+    board[i] = []
+  }
+  board.forEach(row => {
+    for (let j = 0; j < sideLength; j++) {
+      row[j] = 0
+    }
+  })
+  return board
+}
+// initializes a gameboard of 25 cells - NOTE: this can be altered (by user?)
+let board = createBoard(5)
+const side = board.length
+
+// function takes in a non-zero number of cells you want to select
+// and turns that many, randomly selected indices in the sub`board` to `1`
+const selectCells = num => {
+  for (let i = 0; i <= num; i++) {
+    const row = randomInt(side)
+    const cell = randomInt(side)
+    board[row][cell] = 1
+  }
+}
```

Comment:
The commenting throughout the code is great. It makes it super easy to follow. I like that you are just randomly selecting cells to make alive to begin. Seems like a good starting point before switching to having the user select the cells.

Reply:
Thanks!